src/core/stack.js

```diff
@@ -73,6 +73,10 @@ export default class Stack {
 
         if(stack_arguments[0].region && stack_arguments[0].region !== undefined && stack_arguments[0].region !== "us") {
             this.config['host'] = stack_arguments[0].region+"-"+"cdn.contentstack.com";
+            this.config["live_preview"]["host"] =
+                stack_arguments[0].region +
+                "-" +
+                "rest-preview.contentstack.com";
```

Comment:
If it's hard coded, we can write somewhere and use it multiple places. @harshithad0703 

Like it used in 
config as well as stack

Reply:
after making this change, the test are failing